### ISSUE TITLE
Problem with server

### ISSUE THREAD

**Issue by TRLKO** [Exortions/skyblock-2.0#65]
Hi, since I was very interested in this project I tried to download and compile the code. In general, the process itself went without any problems, but it is ineffective to fire up the server. When I start the skyblock plugin, I have no way to join the server where the error is "Timed out". At the moment when I fire up all plugins from the dependencies folder it all works and it is possible to join the server, unfortunately when I add the main plugin - skyblock everything does not work again. I tried joining and reloading the server and by complete restarts. 

Below I will post pictures and all the necessary information:
Java: 8 https://www.openlogic.com/openjdk-downloads
Client version: 1.8.9
Server version: SPIGOT-1.8.8-R0.1-SNAPSHOT-LATEST.JAR (https://getbukkit.org/get/hNiHm0tuqAg1Xg7w7zudk63uHr0xo48D)
Version downloaded from GIT: Latest commit 45f97ad - the latest that was available
Images: 
![Bez nazwy](https://user-images.githubusercontent.com/16487891/209785960-0c1aa1f4-c85b-4f42-80d4-4f31925ed555.png)
![Bez nazwy2](https://user-images.githubusercontent.com/16487891/209785966-d295b42a-2224-4cc3-ba11-9a6acf97a7dd.png)
![Bez nazwy3](https://user-images.githubusercontent.com/16487891/209785974-81a630d0-3fc8-4424-b44c-3aebde33142e.png)

If I can still somehow help with the reproduction please feel free to write.

**Comment by Exortions** [comment]
does anything happen after "Successfully initialized bazaar"? Or do the logs stop there?

**Comment by TRLKO** [comment]
Logs stop over there after bazaar

**Comment by Jason54jg** [comment]
I received the same error it is not possible to download the dependencies for the bazaar (so I took the dependencies myself and put them in the plugins folder and it worked)

**Comment by TRLKO** [comment]
Dear Jason, I took all Skyblock folder to plugins directory at the beginning. This is not an issue in my opinion. Bellow please find some screens.

Regardas

PS: Green information Pernamętnie.... its in translate that Successfully removed all npc from the map. This is message from Citizens plugin witch is by default translate to my native language. 

![Bez nazwy-1](https://user-images.githubusercontent.com/16487891/209916853-859d739d-3419-401a-a849-d5504ad6b6f9.png)
![Bez nazwy2-1](https://user-images.githubusercontent.com/16487891/209916857-5df605e9-6b09-4f21-8221-6ece244b0c0c.png)

**Comment by Exortions** [comment]
does `<plugins folder>/Skyblock/bazaar/items.json` exist?

**Comment by TRLKO** [comment]
@Exortions sorry for delay in response. Yes! Everything is exist - I pull right now the newest version and still I have this issue ehhh. 
<img width="756" alt="Zrzut ekranu 2022-12-30 o 13 51 21" src="https://user-images.githubusercontent.com/16487891/210072106-9bc93902-c761-4b9b-9800-fb971457136c.png">
<img width="738" alt="Zrzut ekranu 2022-12-30 o 13 48 29" src="https://user-images.githubusercontent.com/16487891/210072112-9e8cadfd-8628-48e0-9bff-1063a8972471.png">

**Comment by Exortions** [comment]
Can you send me your full logs (Everything)

**Comment by TRLKO** [comment]
Yes sure:

[latest.log](https://github.com/Exortions/skyblock-2.0/files/10325766/latest.log)

**Comment by Exortions** [comment]
I just made some changes to the code, can you please re-compile the latest commit?

https://github.com/Exortions/skyblock-2.0/commit/a2c054e77794ed1e028d9c13cf8d9cd3783abbe4

**Comment by Exortions** [comment]
@TRLKO Any updates?

**Comment by TRLKO** [comment]
Hi at this moment I finish new year party. In few hours I send feedback and edit this post. I don’t have time @Exortions

**Comment by TRLKO** [comment]
I managed to solve the problem, which turned out to be trivial. Generally, the problem comes up when trying to run the sponge replacers module. At that particular moment, the server starts using 100% of the CPU and the minecraft console window stutters for about 2/3 minutes. The problem, as it turned out, was the insufficient amount of ram in the startup line (1024). When I increased it to a value of 4096 the server started without a problem. I apologize for not checking such an obvious thing at the very beginning.
